core/src/main/java/com/opensymphony/xwork2/interceptor/DefaultResourceIsolationPolicy.java

```diff
@@ -0,0 +1,55 @@
+package com.opensymphony.xwork2.interceptor;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class DefaultResourceIsolationPolicy implements ResourceIsolationPolicy {
+
+    private List<String> exemptedPaths = new ArrayList<String>();
+
+    public List<String> getExemptedPaths(){
+        return this.exemptedPaths;
+    }
+
+    public void setExemptedPaths(List<String> paths){
+        this.exemptedPaths = paths;
+    }
```

Comment:
The exempted paths should be a property of the mitigation itself (the FetchMetadataInterceptor) and not of the individual resource isolation policies, that is: we check the exemptions in the interceptor and only if they aren't matched we call the policy. That way we can get rid of this field and its setter/getter. This implies removing the `if` statement where we currently check for this in the `isRequestAllowed` method
```suggestion
```